Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -247,24 +247,24 @@ public extension Sequence where Element: Equatable {
     ///        ["h", "e", "l", "l", "o"].contains(["l", "o"]) -> true
     ///
     /// - Parameter elements: array of elements to check.
-    /// - Returns: true if array contains all given items.
+    /// - Returns: true if sequence contains all given items.
     /// - Complexity: _O(m·n)_, where _m_ is the length of `elements` and _n_ is the length of this sequence.
     func contains(_ elements: [Element]) -> Bool {
```

Comment:
Was this supposed to be changed as well?

Reply:
Probably it's a leftover from the suggestion, I'll try to clean it up ASAP